packages/front-end/hooks/useSDKConnections.ts

```diff
@@ -1,8 +1,37 @@
 import { SDKConnectionInterface } from "back-end/types/sdk-connection";
+import { WebhookInterface } from "back-end/types/webhook";
 import useApi from "./useApi";
 
-export default function useSDKConnections() {
-  return useApi<{
+interface UseSDKConnectionsOptions {
+  includeWebhooks?: boolean;
+}
+
+export default function useSDKConnections(
+  options: UseSDKConnectionsOptions = {}
+) {
+  const { includeWebhooks = false } = options;
+
+  const connectionsResponse = useApi<{
     connections: SDKConnectionInterface[];
   }>(`/sdk-connections`);
+
+  const webhooksResponse = useApi<{
+    webhooks: Record<string, WebhookInterface[]>;
+  }>(`/sdk-connections/webhooks`, { shouldRun: () => includeWebhooks });
```

Comment:
This feels like it should either be a separate hook, or we should merge the API calls and just have `GET /sdk-connections` optionally return back webhooks as well.

Reply:
moved to separate hook